.github/workflows/build_idf_examples.yml

```diff
@@ -24,5 +24,5 @@ jobs:
           pip install idf-component-manager==1.5.2 idf-build-apps==2.4.3 --upgrade
           python .github/ci/override_managed_component.py esp_tinyusb device/esp_tinyusb ${IDF_PATH}/examples/peripherals/usb/device/tusb_*
           cd ${IDF_PATH}
-          idf-build-apps find --path examples/peripherals/usb/device/ --recursive --target all --manifest-file examples/peripherals/.build-test-rules.yml
-          idf-build-apps build --path examples/peripherals/usb/device/ --recursive --target all --manifest-file examples/peripherals/.build-test-rules.yml
+          idf-build-apps find --path examples/peripherals/usb/device/ --recursive --target all --manifest-file examples/peripherals/.build-test-rules.yml --build-dir build_@t_@w --work-dir @f_@t_@w
```

Comment:
I started to see this problem (when IDF example is built in the same directory for different targets in CI) yesterday. Did something change with the component manager? Or why this has not been an issue until now?

Reply:
We first want to fix it here so we have passing CI in esp-usb.

We use fixed versions of idf-component-manager and idf-build-apps. So the only change can be in esp-idf itself